foundations/introduction/motivation_and_mindset.md

```diff
@@ -74,6 +74,7 @@ As technology advances, we have seen some incredible tools emerge that can help
 While these tools are amazing, learners should be aware of the impacts that using such tools can have on core competency. David Humphrey, a computer science professor, [wrote about ChatGPT and its potentially negative impacts on core learning.](https://blog.humpd.org/cheatgpt/) It is a good read about the pitfalls of using generative AI in an educational context.
 
 For learners that are new to programming, tools like ChatGPT or Github Copilot can result in some blindspots and gaps in your knowledge.
+
```

Comment:
I think this needs to be numbered properly.

Reply:
Oh, I see! Thank you so much for letting me know. It's my first time knowing about Markdown.